CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+## 1.29.1
```

Comment:
can you add this (from https://github.com/aws/amazon-ecs-agent/pull/2093):
Bug: update amazon-vpc-cni-plugins to include a fix (https://github.com/aws/amazon-vpc-cni-plugins/commit/fa071b269ea57644ab566cc915698c61369e158f) that allows awsvpcTrunking to work without ec2-net-utils.